elastic/filebeat/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ES_VERSION
```

Comment:
Can you change this to be a harcoded constant instead? We're probably not going to be dynamically changing the elastic version on a regular basis.